contracts/src/v0.8/ccip/pools/USDC/USDCTokenPool.sol

```diff
@@ -85,15 +92,19 @@ contract USDCTokenPool is TokenPool, ITypeAndVersion {
     if (transmitterVersion != SUPPORTED_USDC_VERSION) revert InvalidMessageVersion(transmitterVersion);
     uint32 tokenMessengerVersion = tokenMessenger.messageBodyVersion();
     if (tokenMessengerVersion != SUPPORTED_USDC_VERSION) revert InvalidTokenMessengerVersion(tokenMessengerVersion);
+    if (cctpMessageTransmitterProxy.i_transmitter() != transmitter) revert InvalidTransmitterInProxy();
 
     i_tokenMessenger = tokenMessenger;
-    i_messageTransmitter = transmitter;
+    i_messageTransmitterProxy = cctpMessageTransmitterProxy;
     i_localDomainIdentifier = transmitter.localDomain();
     i_token.safeIncreaseAllowance(address(i_tokenMessenger), type(uint256).max);
     emit ConfigSet(address(tokenMessenger));
   }
 
   /// @notice Burn the token in the pool
+  /// @notice Outgoing messages (burns) do not require any changes since they are routed
```

Comment:
`do not require any changes` only makes sense in the context of this PR. This comment will live here forever. Please revise the text to make it make sense in a vacuum.